src/test.cc

```diff
@@ -305,9 +305,9 @@ bool RunIndexTests(const std::string& filter_path, bool enable_update) {
   SetTestOutputMode();
 
   std::string clang_version = GetClangVersion();
-  if (clang_version != "6.0.0" && clang_version != "6.0.1") {
+  if (clang_version != "7.0.0") {
```

Comment:
Official version is now 7, not 7.0.0.

Reply:
Are we sure that's what's reported by libclang as well?